### DISCUSSION TITLE
How to I turn ExpressivePixels source code into an executable?

### DISCUSSION THREAD

**Discussion by TrickyToad** [community/community#110487] [Programming Help] [answered]
For whatever reason, Microsoft removed expressive pixels from the Microsoft Store. However, the source code IS still available.
I was wondering if I could get it into an executable or something? I'm not sure what to do with the files provided.
The link is here: https://github.com/microsoft/ExpressivePixels
Any help or ideas would be greatly appreciated!

**Comment by gbpii** [comment, marked as answer]
Check this link: (do not forget to answer if it works)
https://apps.microsoft.com/detail/9mtc56w1rxqh?rtc=1&hl=en-us&gl=US

There is a web page called "Online link generator for Microsoft Store" it will give you app loader for offline installation.

**Reply by TrickyToad** [reply]
Holy crap man! Thanks! You're amazing!

**Reply by gbpii** [reply]
You are welcome! Mark question as answered if you want.
